src/vnode.ts

```diff
@@ -1,4 +1,12 @@
 import {Hooks} from './hooks';
+import {AttachData} from './helpers/attachto'
+import {VNodeStyle} from './modules/style'
+import {On} from './modules/eventlisteners'
+import {Attrs} from './modules/attributes'
+import {Classes} from './modules/class'
+import {Props} from './modules/props'
+import {Dataset} from './modules/dataset'
+import {Hero} from './modules/hero'
```

Comment:
Now pretty much all of the types are exported in their own modules and imported to be used here. I find this cleaner, I guess.